README.md

```diff
@@ -1,10 +1,29 @@
 [![Neon](https://github.com/neondatabase/neon/assets/11527560/f15a17f0-836e-40c5-b35d-030606a6b660)](https://neon.tech)
 
 
-
 # Neon
 
-Neon is a serverless open-source alternative to AWS Aurora Postgres. It separates storage and compute and substitutes the PostgreSQL storage layer by redistributing data across a cluster of nodes.
+<div align="center">
+  <p>Neon is a serverless open-source alternative to AWS Aurora Postgres. It separates storage and compute and substitutes the PostgreSQL storage layer by redistributing data across a cluster of nodes.</p>
+  
+  [![Number of GitHub issues that are open](https://img.shields.io/github/issues/neondatabase/neon)](https://github.com/neondatabase/neon/issues)
+  [![Number of GitHub pull requests that are open](https://img.shields.io/github/issues-pr-raw/neondatabase/neon)](https://github.com/neondatabase/neon/pulls)
+  [![Contributors](https://img.shields.io/github/contributors/neondatabase/neon)](https://github.com/neondatabase/neon/graphs/contributors)
+  [![Latest Release](https://img.shields.io/github/v/release/neondatabase/neon)](https://github.com/neondatabase/neon/releases)
+  [![Number of GitHub stars](https://img.shields.io/github/stars/neondatabase/neon)](https://github.com/neondatabase/neon)
+  [![Forks](https://img.shields.io/github/forks/neondatabase/neon)](https://github.com/neondatabase/neon/network/members)
+  [![Issues Closed](https://img.shields.io/github/issues-closed/neondatabase/neon)](https://github.com/neondatabase/neon/issues?q=is%3Aissue+is%3Aclosed)
+  [![Pull Requests Closed](https://img.shields.io/github/issues-pr-closed/neondatabase/neon)](https://github.com/neondatabase/neon/pulls?q=is%3Apr+is%3Aclosed)
+  [![Follow us on X, formerly Twitter](https://img.shields.io/twitter/follow/neondatabase)](https://x.com/neondatabase)
+  [![Let's Chat on Discord](https://img.shields.io/discord/1176467419317940276)](https://discord.gg/GsWmAzVuAn)
```

Comment:
Using the discord redirect insures that if the link needs to be updated, it only needs to be updated in one place.

```suggestion
  [![Let's Chat on Discord](https://img.shields.io/discord/1176467419317940276)](https://neon.tech/discord)
```